src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -118,6 +118,9 @@ void OTARequestor::OnQueryImageResponse(void * context, const QueryImageResponse
 {
     LogQueryImageResponse(response);
 
+    uint32_t currentSoftwareVersion;
+    chip::app::Clusters::Basic::Attributes::SoftwareVersion::Get(0, &currentSoftwareVersion);
```

Comment:
Should check for error, no?  Otherwise we can end up working with an un-initialized value...

Reply:
updated to krootendpointid

Reply:
done